src/ascii.rs

```diff
@@ -45,6 +45,7 @@ pub fn get_ascii(os: &str) -> (String, u16) {
         "almalinux" => ALMA,
         "android" => ANDROID,
         "garuda" => GARUDA,
+        "linux" => LINUX,
```

Comment:
Ignoring this has nothing to do with the PR, I would gate this behind a config option defaulted to off simply to draw attention to any missing ASCII's for more bug reports to add them. Once it gets reported they can get advised they use the config option as an alternative while it gets added.